akka-projection-kafka/src/main/scala/akka/projection/kafka/internal/KafkaSourceProviderImpl.scala

```diff
@@ -37,7 +40,7 @@ import org.apache.kafka.common.record.TimestampType
  * INTERNAL API
  */
 @InternalApi private[projection] object KafkaSourceProviderImpl {
-  private[kafka] type ReadOffsets = () => Future[Option[GroupOffsets]]
+  private[kafka] type ReadOffsets = () => Future[Option[MergeableOffset[JLong]]]
```

Comment:
And then there is this, we need to use `JLong` (java.lang.Long) everywhere. If we use the `scala.Long`, Java sees it as `Object` and code don't compile.